core/domain/exp_services_test.py

```diff
@@ -5648,3 +5732,3136 @@ def test_when_few_state_stats_models_are_missing_for_old_exps(self):
+            'property_name': 'next_content_id_index',
+            'old_value': 3,
+            'state_name': 'Introduction',
+            'new_value': 4
+        }), exp_domain.ExplorationChange({
+            'cmd': 'edit_state_property',
+            'property_name': 'solution',
+            'old_value': None,
+            'state_name': 'Introduction',
+            'new_value': {
+                'correct_answer': 'Hello Aryaman!',
+                'explanation': {
+                    'content_id': 'solution',
+                    'html': '<p>Explanation.</p>'
+                },
+                'answer_is_exclusive': False
+            }
+        })]
+
+        exp_services.update_exploration(
+            self.owner_id, self.EXP_0_ID, change_list,
+            'Added answer groups and solution')
+
+        # Changes to the properties unrelated to the solutions.
+        change_list_2 = [exp_domain.ExplorationChange({
+            'new_value': {
+                'content_id': 'content',
+                'html': '<p>This is the first state.</p>'
+            },
+            'state_name': 'Introduction',
+            'old_value': {
+                'content_id': 'content',
+                'html': ''
+            },
+            'cmd': 'edit_state_property',
+            'property_name': 'content'
+        }), exp_domain.ExplorationChange({
+            'new_value': [{
+                'hint_content': {
+                    'content_id': 'hint_3',
+                    'html': '<p>Hint 1.</p>'
+                }
+            }, {
+                'hint_content': {
+                    'content_id': 'hint_4',
+                    'html': '<p>This is a first hint.</p>'
+                }
+            }],
+            'state_name': 'Introduction',
+            'old_value': [{
+                'hint_content': {
+                    'content_id': 'hint_3',
+                    'html': '<p>Hint 1.</p>'
+                }
+            }],
+            'cmd': 'edit_state_property',
+            'property_name': 'hints'
+        }), exp_domain.ExplorationChange({
+            'new_value': 5,
+            'state_name': 'Introduction',
+            'old_value': 4,
+            'cmd': 'edit_state_property',
+            'property_name': 'next_content_id_index'
+        }), exp_domain.ExplorationChange({
+            'new_value': {
+                'content_id': 'content',
+                'html': '<p>Congratulations, you have finished!</p>'
+            },
+            'state_name': 'End',
+            'old_value': {
+                'content_id': 'content',
+                'html': ''
+            },
+            'cmd': 'edit_state_property',
+            'property_name': 'content'
+        })]
+        exp_services.update_exploration(
+            self.owner_id, self.EXP_0_ID, change_list_2,
+            'Changed Contents and Hint')
+
+        # Changes to the solutions and the related properties
+        # to check for mergeability.
```

Comment:
What are "related properties"? Please list them.

Reply:
Related here means the properties which are affected by or affecting the solutions of the property. I have changed the statement so that it can be more clear. Still, do I need to mention the list of the properties?